### ISSUE TITLE
Set seed

### ISSUE THREAD

**Issue by mo26-web** [DLR-RM/stable-baselines3#1098]
Hi
when I re-compile my algorithm, I found that set seed attribute is not exist anymore. Example:

model = PPO2(
    CustomActorCriticPolicy,
    env,
    #policy_kwargs=policy_kwargs,
    learning_rate=0.001,
      # progress decreases from 1 to 0 -> lr decreasesb from 0.0021 to 0
    n_steps=256, 
    batch_size=256, 
    n_epochs=100, 
    gamma=0.001,
    tensorboard_log="log_25",  # define the tensorboard log location
    seed=0,
    
)

I think new changes in source and core my cause this problem. Please show me proper way to solve this.

**Comment by qgallouedec** [comment]
PPO2 is not part of stable-baselines3 but stable-baselines.

The right way to seed the agent is

```python
from stable_baselines3 import PPO

PPO("MlpPolicy", "CartPole-v1", seed=0)
```
